source/dstats/distrib.d

```diff
@@ -118,7 +118,13 @@ enum SQ2PI = 2.50662827463100050241576528481104525300698674060993831662992;
 version(unittest) {
     import std.stdio, std.random;
 
-    alias std.math.approxEqual ae;
+    static if (__VERSION__ < 2096)
+        alias approxEqual = std.math.approxEqual;
+    else
+        bool approxEqual(T, U, V)(T lhs, U rhs, V maxRelDiff = 1e-2, V maxAbsDiff = 1e-5)
+        {
+            return std.math.isClose(lhs, rhs, 1e-2, 1e-5); // mimic old sloppy approxEqual for now
+        }
```

Comment:
why not put this somewhere central? Instead of 2 copies.

Reply:
I will, thanks.